src/integration_tests/schemas/test_integration_unhappy_path.py

```diff
@@ -0,0 +1,170 @@
+from unittest import TestCase
+import pytest
+from src.app.config.config_factory import config
+from src.integration_tests.helpers.integration_helpers import (
+    cleanup,
+    generate_headers,
+    pubsub_setup,
+    pubsub_teardown,
+    setup_session,
+    inject_wait_time,
+)
+from src.integration_tests.helpers.pubsub_helper import schema_pubsub_helper
+from src.test_data.schema_test_data import test_survey_id
+from src.test_data.shared_test_data import test_schema_subscriber_id
+
+class E2ESchemaIntegrationUnhappyPaths(TestCase):
+    session = None
+    headers = None
+    invalid_survey_id = "nonexistent_survey"
+    invalid_data = {"invalid": "data"}
+    invalid_token_headers = None
+
+    @classmethod
+    def setup_class(cls) -> None:
+        cleanup()
+        pubsub_setup(schema_pubsub_helper, test_schema_subscriber_id)
+        inject_wait_time(3) 
+        cls.session = setup_session()
+        cls.headers = generate_headers()
+        cls.invalid_token_headers = {"Authorization": "Bearer invalid_token"}
+
+    @classmethod
+    def teardown_class(cls) -> None:
+        cleanup()
+        inject_wait_time(3) # Allow all messages to process
+        pubsub_teardown(schema_pubsub_helper, test_schema_subscriber_id)
+
+    @pytest.mark.order(1)
+    def test_post_schema_unauthorized(self):
+        """
```

Comment:
could docstring follow layout similar to James PR
 """
        Test for /v1/dataset_metadata endpoint without passing survey_id parameter
        
        - Get request to retrieve metadata without passing survey_id
        - Assert status code is 400
        - Checks the error message in the response
        """